README.md

```diff
@@ -85,6 +85,8 @@ end
 ```julia
 julia> using PortAudio, SampledSignals, LibSndFile
```

Comment:
```suggestion
julia> import LibSndFile # must be in Manifest for FileIO.save to work
julia> using PortAudio: PortAudioStream
julia> using SampledSignals: s
julia> using FileIO: save
```
To help newcomers understand such examples, I think it is preferable to show exactly which methods are used from which packages.